r/NEWS.md

```diff
@@ -19,6 +19,10 @@
 
 # arrow 18.1.0.9000
 
+- Added missing dplyr binding for atan (#44953)
+- Added bindings for hyperbolig trig functions sinh, cosh, tanh, asinh, acosh, and tanh (#44953)
+- Added bindings for expm1 (#44953)
```

Comment:
It doesn't _really_ matter but I wonder if collapsing these into one bullet (since there's one issue) would be better?

Reply:
One bullet is fine, I'll change it.